packages/portalnetwork/src/networks/contentLookup.ts

```diff
@@ -25,6 +25,7 @@ export class ContentLookup {
   private finished: boolean
   private content: ContentLookupResponse
   private pending: Set<NodeId>
+  private queuedPeers: Set<NodeId>
```

Comment:
How do you remove `queuedPeers`? It seems like it will grow forever

Reply:
`queuedPeers` should always be a full list of all the peers ever added to the `lookupPeers` heap, so that we can avoid re-adding a peer to the heap after it has been popped

Reply:
ok thanks for the clarification! great fix